lib/tessel/deploy.js

```diff
@@ -54,6 +54,26 @@ Tessel.prototype.deployScript = function(opts) {
   });
 };
 
+Tessel.prototype.restartScript = function(opts) {
+  var self = this;
+  var isPush = opts.type === 'flash';
+  var filepath = isPush ? PUSH_PATH : RUN_PATH;
+
+  return new Promise(function(resolve, reject) {
+    actions.findProject(opts.entryPoint).then(function(project) {
```

Comment:
I think we can remove the entrypoint argument and not check for a local folder before restarting on the remote device.